spotbugs/etc/findbugs.xml

```diff
@@ -1254,4 +1256,6 @@
           <BugPattern abbrev="NP" type="NP_METHOD_PARAMETER_TIGHTENS_ANNOTATION" category="STYLE" />
           <BugPattern abbrev="NP" type="NP_METHOD_PARAMETER_RELAXING_ANNOTATION" category="STYLE" deprecated="true" />
           <BugPattern abbrev="EOS" type="EOS_BAD_END_OF_STREAM_CHECK" category="CORRECTNESS" />
+          <BugPattern abbrev="MC" type="MC_OVERRIDABLE_METHOD_CALL_IN_CONSTRUCTOR" category="BAD_PRACTICE" />
+          <BugPattern abbrev="MC" type="MC_OVERRIDABLE_METHOD_CALL_IN_CLONE" category="BAD_PRACTICE" />
```

Comment:
I think `BAD_PRACTICE` is acceptable, but [this page](https://wiki.sei.cmu.edu/confluence/display/java/MET05-J.+Ensure+that+constructors+do+not+call+overridable+methods) pointed that code with this problem may lead to vulnerability then the [`MALICIOUS_CODE` cateogry](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#malicious-code-vulnerability-malicious-code) is also our option.

Reply:
I absolutely agree with you, this bug could land in any of these categories. Now I feel that MALICIOUS_CODE is maybe better than BAD_PRACTICE. Should I change it? Or maybe should we wait for others to vote, e.g. @ThrawnCA?